package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "project",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "eject": "expo eject"
+  },
+  "dependencies": {
+    "@react-navigation/native": "^6.0.8",
+    "@react-navigation/native-stack": "^6.5.0",
+    "expo": "~44.0.0",
+    "expo-linear-gradient": "~11.0.3",
+    "expo-status-bar": "~1.2.0",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-native": "0.64.3",
+    "react-native-elements": "^3.4.2",
+    "react-native-image-gradient": "^2.2.5",
+    "react-native-vector-icons": "^9.1.0",
+    "react-native-web": "0.17.1"
```

Comment:
Are you planning to publish the app on the web too? Nice!

Reply:
The 2 libs are useful to launch on the navigator for preview if you don't have any simulator. It will maybe not stay available in the future.

Reply:
it could be nice as a POC for RNW too, worth keeping it and supporting it maybe.